pkg/apis/policy/v1alpha1/trustroot_validation.go

```diff
@@ -99,8 +99,8 @@ func (remote *Remote) Validate(ctx context.Context) (errors *apis.FieldError) {
 }
 
 func (sigstoreKeys *SigstoreKeys) Validate(ctx context.Context) (errors *apis.FieldError) {
-	if len(sigstoreKeys.CertificateAuthorities) == 0 {
-		errors = errors.Also(apis.ErrMissingField("certificateAuthority"))
+	if len(sigstoreKeys.CertificateAuthorities) == 0 && len(sigstoreKeys.TimeStampAuthorities) == 0 {
```

Comment:
FYI, we also need to update that assumption here when we're pulling things from TUF repos / remotes once we add support for getting TSA keys from there:
https://github.com/sigstore/policy-controller/blob/main/pkg/reconciler/trustroot/trustroot.go#L248